release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -461,6 +461,13 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
 [id="ocp-4-11-olm"]
 === Operator lifecycle
 
+[id="ocp-4-11-olm-fbc"]
+==== File-based catalog format
+
+The default Red Hat-provided Operator catalogs for {product-title} 4.11 ships in the file-based catalog format. {product-title} 4.6 through 4.10 shipped in the SQLite database format. File-based catalogs are the latest iteration of the catalog format in Operator Lifecycle Manager (OLM). It is a plain text-based (JSON or YAML) and declarative config evolution of the earlier SQLite database format. Cluster administrators and users will not see any change to their install workflows and Operator consumption with the new catalog format.
```

Comment:
> 🚫 ship verb
Do not use to indicate that something is included in or with a product. Use "include" or "included".

> Avoid using parentheses in general text because text in parentheses is decreased in importance.

> ! config adjective, noun
Do not use to mean "configuration" except in command, parameter, or file names. For example, "Update the configuration by editing the config.txt file."
([Ignition config](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#ignition-config) and config map are the only exception that I'm aware of.)

Reply:
Good to know about Ignition and MCP, I'll keep that in mind for reviews as well. Thanks!